beetsplug/check.py

```diff
@@ -245,7 +245,13 @@ def add(item):
             log.debug(
                 u'adding checksum for {0}'.format(displayable_path(item.path))
             )
-            set_checksum(item)
+            try:
+                set_checksum(item)
+            except FileNotFoundError as ex:
+                log.warning(u'{} {}: {}'.format(
+                    colorize('text_warning', u'WARNING'), 'No such file',
```

Comment:
The beet logger already prints the `WARNING` prefix. We can omit it here.

Reply:
To be clear, should I change this line to this?

    log.warning(u'{}: {}'.format('No such file', displayable_path(item.path))

I can see the same pattern on lines 260 and 293, do those also need fixing?

Reply:
Ignore my previous comment. You’re right on following the pattern. I was looking at the log output from the tests. But that is not what the user sees. You can keep it like it is. (You just need to update the tests)